composer/core/state.py

```diff
@@ -27,6 +27,7 @@
 from composer.core.time import Time, Timestamp, TimeUnit
 from composer.devices import Device
 from composer.utils import batch_get, batch_set, dist, ensure_tuple, get_composer_env_dict, is_model_deepspeed
+from composer.utils.misc import using_torch_2_0
```

Comment:
can we rename this to `using_torch_2` since presumably we gate for >= 2.0?

Reply:
done